examples/flutter_gallery/lib/demo/shrine/shrine_home.dart

```diff
@@ -306,12 +304,8 @@ class _ShrineHomeState extends State<ShrineHome> {
   void showOrderPage(Product product) {
     final Order order = _shoppingCart[product] ?? new Order(product: product);
     final Completer<Order> completer = new Completer<Order>();
-    final Key productKey = new ObjectKey(product);
-    final Set<Key> mostValuableKeys = new HashSet<Key>();
-    mostValuableKeys.add(productKey);
     Navigator.push(context, new ShrineOrderRoute(
       order: order,
-      settings: new RouteSettings(mostValuableKeys: mostValuableKeys),
```

Comment:
i wonder if maybe we should remove most valuable keys in general. I had originally added them because I assumed that it'd be easier to just have one tag on the target page and not know what the source tag would be, but in practice in every demo it's been easy to know what the source tag would be.